src/server.ts

```diff
@@ -27,6 +27,29 @@ import ConfigResolver, {
 
 } from 'vscode-config-resolver';
 
+interface IWorkspaceSettings {
+	browsers: string[];
+	ignore: string[];
+	ignoreFiles: string[];
+	messageLevel: string;
+	run: string;
+}
+
+type IBrowsersList = string[];
+
+interface IBrowsersListCache {
+	[key: string]: IBrowsersList;
+}
+
+interface IProblem {
+	feature: number;
+	featureData: Object;
+	usage: {
```

Comment:
JFYI: It is best to avoid describing nested properties in interfaces. Most likely, in the future you will want to use this field and need to describe it in more detail. You can put it in a separate type.

Reply:
Well, I hate them too, however, I only wanted to resolve the `: any`s around the file. So... for me this would be enough for now. This can be further improved when needed.